resources/js/components/dashboard/ControlGroup.vue

```diff
@@ -1,7 +1,19 @@
 <template>
-  <div class="control-group container">
-    <h1>Control Group</h1>
+<div class="control-group container">
+  <div class="row">
+    <div class="col-8 offset-2 text-center">
+      <div class="control-group__body">
+        <i class="control-group__icon fas fa-user"></i>
+        <div class="control-group__content">
```

Comment:
Unless we're going to reuse this class somewhere else, I would advise against creating a class for margins when you could use a margin[ Bootstrap class](https://getbootstrap.com/docs/4.0/utilities/spacing/#notation) to do so.

Other than that, your changes are a definite improvement!

Reply:
dannnng i forgot about those. Okay thank you!